FertigMelder/module.php

```diff
@@ -140,5 +79,23 @@ public function RequestAction($Ident, $Value) {
 		}
 	}
 
+	public function MessageSink($TimeStamp, $SenderID, $Message, $Data)
```

Comment:
Hast du das getestet? Funktioniert das wie in der Dokumentation beschrieben?

Reply:
Das dürfte nicht der Dokumentation entsprechen. Was passiert, wenn während des Warteintervalls mehrmals der Wert unterhalb des Grenzwertes aktualisiert wird?